examples/drivenet/drivenet_open_loop.ipynb

```diff
@@ -0,0 +1,354 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DriveNet Open Loop Evaluation\n",
+    "\n",
+    "**Note: this notebook assumes you've already run the [training notebook](./drivenet_train.ipynb) and stored your model successfully.**\n",
+    "\n",
+    "### What is open loop evaluation?\n",
+    "In open-loop evaluation we evaluate our model prediction as we follow the annotated ground truth.\n",
```

Comment:
I'm actually not sure now whether its open loop or open-loop - but could u make it consistent?